File: note-keeper-service/src/main/java/xyz/crearts/note/keeper/model/Attachment.java
package xyz.crearts.note.keeper.model;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Attachment {
    private String id;
    private String parentId;
    private String parentType;
    private String name;
    private long size;
    private String type;
    private String url;
    private LocalDateTime uploadedAt;
}

File: android/app/src/main/java/xyz/crearts/notekeeper/data/model/Attachment.kt
package xyz.crearts.notekeeper.data.model

data class Attachment(
    val id: String,
    val parentId: String,
    val parentType: String,
    val name: String,
    val size: Long,
    val type: String,
    val url: String,
    val uploadedAt: String
)
